floyd/cli/run.py

```diff
@@ -279,8 +276,7 @@ def run(ctx, cpu, gpu, env, message, data, mode, open_notebook, follow, tensorbo
         sys.exit(2)
 
     # Create module
-    default_name = 'input' if len(data_ids) <= 1 else None
-    module_inputs = [{'name': get_data_name(data_str, default_name),
+    module_inputs = [{'name': get_data_name(data_str),
```

Comment:
On second thoughts, this function is so simple now this can be called directly:

```suggestion
    module_inputs = [{'name': data_str.split(':')[1],
```